source/updateCheck.py

```diff
@@ -237,28 +237,50 @@ def executePendingUpdate():
 		_executeUpdate(updateTuple[0])
 
 
-def _executeUpdate(destPath):
+def _executeUpdate(destPath: str) -> None:
+	"""Execute the update process.
+
+	:param destPath: The path to the update executable.
+	:raises ValueError: If destPath is empty or not given.
+	"""
 	if not destPath:
-		return
+		raise ValueError("destPath must be a non-empty string")
```

Comment:
I'm concerned this will throw an unrecoverable error when updating from a version of NVDA that  doesn't send `destPath`